pylint/checkers/variables.py

```diff
@@ -2430,6 +2423,17 @@ def _check_unpacking(self, inferred, node, targets):
                 args=(_get_unpacking_extra_info(node, inferred),),
             )
 
+    @staticmethod
+    def _get_values_to_unpack(node: nodes.Assign) -> Optional[List]:
```

Comment:
Could you add typing to the List? I guess:
```suggestion
    def _get_values_to_unpack(node: nodes.Assign) -> Optional[List[nodes.AssignName]]:
```
Or:
```suggestion
    def _get_values_to_unpack(node: nodes.Assign) -> Optional[List[nodes.NodeNG]]:
```

Reply:
Good point; I think it is more accurate to use `Optional[List[nodes.NodeNG]]` since the nodes which are unpacked won't necessarily be of type `nodes.AssignName`.
Moreover we can make the function name more accurate (`_nodes_to_unpack`?) & the type of the parameter should be `nodes.NodeNG` since this value originates from the returned value of the `safe_infer` method.